.deployment/barista/Jenkinsfile

```diff
@@ -140,19 +132,28 @@ pipeline {
                 node dist/libs/tools/examples-tools/main.js
                 node dist/libs/tools/barista/main.js
 
-                if $INTERNAL_VERSION ; then
-                  configuration='production-internal'
-                else
+                if $PUBLIC_BUILD ; then
                   configuration='production'
+                else
+                  configuration='production-internal'
                 fi
 
+                cat dist/barista-data/routes.txt
```

Comment:
Was this added for debugging purposes?

Reply:
yea than we can detect faster on the Jenkins if an error occurred.